lib/matplotlib/tests/__init__.py

```diff
@@ -10,6 +10,7 @@
 
 _multiprocess_can_split_ = True
 
+import pyparsing;
```

Comment:
I wanted to add a warning for version 2.1.2 (which has a python3 compatibility bug) and decided not to do that, and ultimately forgot to delete this.